src/App.ts

```diff
@@ -946,6 +964,12 @@ export default class App<AppCustomContext extends StringIndexed = StringIndexed>
       retryReason: event.retryReason,
     };
 
+    if (this.attachFunctionToken) {
+      const { functionExecutionId, functionBotToken } = extractFunctionContext(body);
+      if (functionExecutionId) { context.functionExecutionId = functionExecutionId; }
+      if (functionBotToken) { context.functionBotToken = functionBotToken; }
```

Comment:
Naming is hard, but I think the consensus was to name this `function_bot_access_token` (`functionBotAccessToken`)

I think we should be consistent between the implementation I don't mind changing python and java to this, let me know what you prefer?

Reply:
Fair callout! I'm fine with swapping this out.